packages/config/src/projects/layer2s/parallel.ts

```diff
@@ -2,15 +2,14 @@ import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
 
 import { subtractOne } from '../../common/assessCount'
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
-import { Badge } from '../badges'
 import { orbitStackL2 } from './templates/orbitStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('parallel')
 
 export const parallel: Layer2 = orbitStackL2({
   discovery,
-  badges: [Badge.RaaS.Conduit],
+  badges: [],
```

Comment:
Can we add the relevant badges instead of leaving it empty?

Reply:
they're added via template. i noticed that the explorer link doesnt work so im looking for one rn

Reply:
ah indeed, nvm then

Reply:
but the template field should definitely be "additionalBadges" because like this it is very confusing indeed